src/modules/settings/components/Window.jsx

```diff
@@ -28,12 +28,15 @@ function Page(props) {
 }
 
 function isWindowSmallStandaloneChat() {
-  return window.innerWidth < 600 && window.location.pathname.endsWith('/chat');
+  return (
```

Comment:
you can't do this. site specific logic should be in a youtube/twitch component folder (see emote menu)